tasks/poeditor.js

```diff
@@ -26,15 +23,64 @@ module.exports = function(g) {
                 done();
             });
         }
+        // download
+        else if (data.download) {
+            var downloadMeta = confLanguages(data.download, opts);
+            downloadMeta.project = opts.project;
+            download(downloadMeta, opts, this.async());
+        }
+        // download_all
+        else if (data.download_all) {
+            var done = this.async();
+            getLanguagesToDownload(data.download_all, opts, function(err, languages) {
+              if(err) {
+                done(false);
+                return;
+              }
+
+              opts.languages = _.extend(languages, opts.languages_override || {});
```

Comment:
_.extend(languages, opts.languages_override || {});
you can skip the || {}
.extend handles it.